lib/timer_controller.dart

```diff
@@ -10,6 +10,12 @@ class CountdownController {
   // Called when called `resume` method
   VoidCallback onResume;
 
+  // called to restart the timer
+  VoidCallback onRestart;
+
+  //is timer active
+  bool isCompleted = false;
```

Comment:
This param is not used
Maybe it should be removed

Reply:
I wanted to implement the "isCompleted" to be able to toggle Restart and Resume.
for example:
`_controller.isCompleted ? _controller.restert : _contoller.resume`

Reply:
If you want and can, may you add all functionality? 
That was be really cool!

Reply:
I have added it today, you can check it out.

Reply:
okay, i'll check it!